core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1526,7 +1526,7 @@ class SparkContext(config: SparkConf) extends Logging {
   def addFile(path: String, recursive: Boolean): Unit = {
     val uri = new Path(path).toUri
     val schemeCorrectedPath = uri.getScheme match {
```

Comment:
This path string is only used to create a hadoop Path. Let me confirm: can we create hadoop Path with URI directly?